cmd/cni-metrics-helper/main.go

```diff
@@ -120,15 +131,15 @@ func main() {
 		if err != nil {
 			log.Fatalf("Failed to create publisher: %v", err)
 		}
-		go cw.Start()
+		var publishInterval = pullInterval * 2
+		go cw.Start(publishInterval)
```

Comment:
bit nit, but it is a bit more golang to do `publishInterval := pullInterval * 2` or to just pass `pullInterval * 2` to function

Reply:
Completed in the latest commit

Sorry. My language of choice is Java. My Java-like expressiveness is showing through :)